cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
```

Comment:
I like having the package `import wdl4s.cwl._` but it's just a nice to have.  If @geoffjentry feels strongly otherwise let's do it somewhere else.